src/datanode/src/tests/instance_test.rs

```diff
@@ -342,6 +342,104 @@ pub async fn test_execute_create() {
     assert!(matches!(output, Output::AffectedRows(0)));
 }
 
+#[tokio::test]
+async fn test_rename_table() {
+    let instance = MockInstance::new("test_rename_table_local").await;
+
+    let output = execute_sql(&instance, "create database db").await;
+    assert!(matches!(output, Output::AffectedRows(1)));
+
+    let output = execute_sql_in_db(
+        &instance,
+        "create table demo(host string, cpu double, memory double, ts timestamp, time index(ts))",
+        "db",
+    )
+    .await;
+    assert!(matches!(output, Output::AffectedRows(0)));
+
+    let output = execute_sql_in_db(&instance, "show tables", "db").await;
+    let expect = "\
++--------+
+| Tables |
++--------+
+| demo   |
++--------+\
+"
+    .to_string();
+    check_output_stream(output, expect).await;
+
+    // make sure table insertion is ok before altering table name
+    let output = execute_sql_in_db(
+        &instance,
+        "insert into demo(host, cpu, memory, ts) values ('host1', 1.1, 100, 1000)",
+        "db",
+    )
+    .await;
+    assert!(matches!(output, Output::AffectedRows(1)));
+
+    // rename table
+    let output = execute_sql_in_db(&instance, "alter table demo rename test_table", "db").await;
+    assert!(matches!(output, Output::AffectedRows(0)));
+
+    let output = execute_sql_in_db(&instance, "show tables", "db").await;
+    let expect = "\
++------------+
+| Tables     |
++------------+
+| test_table |
++------------+\
+"
+    .to_string();
+    check_output_stream(output, expect).await;
+
+    // make sure table insertion is ok after altered table name
+    let output = execute_sql_in_db(
+        &instance,
+        "insert into test_table(host, cpu, memory, ts) values ('host2', 2.2, 200, 2000)",
+        "db",
+    )
+    .await;
+    assert!(matches!(output, Output::AffectedRows(1)));
+
+    let output = execute_sql_in_db(&instance, "select * from test_table order by ts", "db").await;
+    let expected = "\
++-------+-----+--------+---------------------+
+| host  | cpu | memory | ts                  |
++-------+-----+--------+---------------------+
+| host1 | 1.1 | 100    | 1970-01-01T00:00:01 |
+| host2 | 2.2 | 200    | 1970-01-01T00:00:02 |
++-------+-----+--------+---------------------+\
+"
+    .to_string();
+    check_output_stream(output, expected).await;
+
+    // restart instance
+    assert!(instance.restart().await.is_ok());
```

Comment:
Please use unwrap instead of `assert!(xxx.is_ok)` so we could get the error message
```suggestion
    instance.restart().await.unwrap()
```